blueboxgroup/ursula
# cleaning rbd_volumes entries

- Cleanup `rbd_volumes` related entries from roles and playbook still referencing it as this is generated automatically.
- ceph.yml playbook is enhanced to check for new backends.

Testing:
- Run ursula with ceph tags to make sure there are no failures 
- Run ceph.yml playbook to make sure it recognizes both the new backend names as well as fail when it sees rbd_volumes in /etc/cinder/cinder.conf on controller nodes.
